src/sardana/tango/core/util.py

```diff
@@ -620,6 +675,13 @@ def prepare_environment(args, tango_args, ORB_args):
             env_name = arg[1:]
             env_val = ORB_args[i + 1]
             os.environ[env_name] = env_val
+
+            # In Tango > 7 the omniORB parameters has a conflict if are
```

Comment:
This workaround is too agressive. It removes all ORB arguments.
Seems like from Tango 8 we don't need this workaround, see [tango-cs#495](https://sourceforge.net/p/tango-cs/bugs/495/)